test/ConstructiveSolidGeometry/CSG_primitives.jl

```diff
@@ -161,4 +161,58 @@ no_translations = (rotation = one(SMatrix{3, 3, T, 9}), translation = zero(Carte
         @test !in(CartesianPoint{Float64}(1,1,3+tol),box_closed_trafo)
         @test in(CartesianPoint{Float64}(1,1,3-tol),box_open_trafo)
     end
+    @testset "RegularPrism" begin
+        prism1 = CSG.RegularPrism(CSG.ClosedPrimitive,r=1f0, hZ = 2f0)
```

Comment:
I prefer the first option. I agree that a default does not make much sense here.

Reply:
Me too. Also because we would get in trouble with other parts of the code like `surfaces` as the number of returned surfaces would depend on the value `N` in the second case.

Could you please adapt to the first option?

Reply:
I also remember that we have aliases for the `RegularPrisms`.
Is it implemented that e.g. calling `HexagonalPrism(...)` would execute the code as if it was `RegularPrism{6}(....)`?

Reply:
No, Hexagonal Prism is for now still an alias (returns struct rightaway). Maybe it does make sense from a user perspective to change that.